src/libs/auth.js

```diff
@@ -64,9 +65,20 @@ authStore.subscribe(async (state, oldState) => {
         )
       )
 
+      const needsUpgrade = _.remove(c => c.labels.saturnVersion === version, clusters)
+      await Promise.all(_.map(({ googleProject, clusterName, machineConfig, jupyterUserScriptUri }) => {
+        const cluster = Leo.cluster(googleProject, clusterName)
+        const clusterOptions = {
+          labels: { 'saturnAutoCreated': 'true', 'saturnVersion': version },
+          machineConfig,
+          jupyterUserScriptUri
+        }
+        return cluster.delete().then(() => cluster.create(clusterOptions))
```

Comment:
This will re-use the same name. Is that safe, or should we generate a new one?

Reply:
I'm not sure, but I think it's safe.

Reply:
I'm not finding any problems.

Reply:
I _think_ you should be just fine.

Reply:
Alternately, if we create one with a new name, we wouldn't have to wait for the delete to finish to start creating it, which might be better.